### ISSUE TITLE
Add Missing "Accessibility Features" Tags In Store Page

### ISSUE THREAD

**Issue by NovaDiablox** [Facepunch/garrysmod-requests#2966]
### Description

Including but not limited to

- [Save Anytime](https://help.steampowered.com/faqs/view/02F5-ACB2-6038-0F36#saving_options)
- [Subtitle Options](https://help.steampowered.com/faqs/view/02F5-ACB2-6038-0F36#subtitles)
- [Camera Comfort](https://help.steampowered.com/faqs/view/02F5-ACB2-6038-0F36#camera_comfort)
- [Custom Volume Controls](https://help.steampowered.com/faqs/view/02F5-ACB2-6038-0F36#volume_controls)
- [Stereo Sound](https://help.steampowered.com/faqs/view/02F5-ACB2-6038-0F36#stereo_sound)
- [Surround Sound](https://help.steampowered.com/faqs/view/02F5-ACB2-6038-0F36#surround_sound)

**Comment by robotboy655** [comment]
Garry's Mod does not have any "Subtitle Options" (i.e. customizing subtitles)

"Camera Comfort" is also not a thing, yes you can disable motion blur, but you cannot disable screen shake.

I am not sure "Surround Sound" is supported.

Going through Steam's wizard for these features ends us up with this:

<img width="388" height="292" alt="Image" src="https://github.com/user-attachments/assets/4931f576-431d-4e52-9268-059be0fa021a" />

Adjustable difficulty (skill convar)
No QTEs, so "Playable without Timed input"
No reliance on colors for important information, so "Color alternatives"

Let me know if you disagree on any of this. It's not currently live, I will likely push it to live with the upcoming update if I don't forget.
